src/PointwiseFunctions/AnalyticData/GrMhd/SpecInitialData.cpp

```diff
@@ -100,47 +100,49 @@ template <size_t ThermodynamicDim>
 template <typename DataType>
 void SpecInitialData<ThermodynamicDim>::VariablesComputer<DataType>::operator()(
     const gsl::not_null<Scalar<DataType>*> specific_internal_energy,
-    const gsl::not_null<Cache*> /*cache*/,
+    const gsl::not_null<Cache*> cache,
     hydro::Tags::SpecificInternalEnergy<DataType> /*meta*/) const {
   const auto& rest_mass_density =
       get<hydro::Tags::RestMassDensity<DataType>>(interpolated_data);
+  const auto& temperature =
+      cache->get_var(*this, hydro::Tags::Temperature<DataType>{});
   const size_t num_points = get_size(get(rest_mass_density));
   for (size_t i = 0; i < num_points; ++i) {
     const double local_rest_mass_density =
         get_element(get(rest_mass_density), i);
-    if (local_rest_mass_density <= density_cutoff) {
-      get_element(get(*specific_internal_energy), i) = 0.;
-    } else {
+    const double local_temperature =
+        get_element(get(temperature), i);
       if constexpr (ThermodynamicDim == 1) {
         get_element(get(*specific_internal_energy), i) =
             get(eos.specific_internal_energy_from_density(
                 Scalar<double>(local_rest_mass_density)));
       } else if constexpr (ThermodynamicDim == 2) {
         get_element(get(*specific_internal_energy), i) =
             get(eos.specific_internal_energy_from_density_and_temperature(
-                Scalar<double>(local_rest_mass_density), Scalar<double>(0.)));
+                Scalar<double>(local_rest_mass_density),
+                Scalar<double>(local_temperature)));
       } else {
         ERROR("Only 1d & 2d EOS is currently supported for SpEC ID");
       }
-    }
   }
 }
 
 template <size_t ThermodynamicDim>
 template <typename DataType>
 void SpecInitialData<ThermodynamicDim>::VariablesComputer<DataType>::operator()(
     const gsl::not_null<Scalar<DataType>*> pressure,
-    const gsl::not_null<Cache*> /*cache*/,
+    const gsl::not_null<Cache*> cache,
     hydro::Tags::Pressure<DataType> /*meta*/) const {
   const auto& rest_mass_density =
       get<hydro::Tags::RestMassDensity<DataType>>(interpolated_data);
+  const auto& temperature =
+      cache->get_var(*this, hydro::Tags::Temperature<DataType>{});
   const size_t num_points = get_size(get(rest_mass_density));
   for (size_t i = 0; i < num_points; ++i) {
     const double local_rest_mass_density =
         get_element(get(rest_mass_density), i);
-    if (local_rest_mass_density <= density_cutoff) {
-      get_element(get(*pressure), i) = 0.;
-    } else {
+    const double local_temperature =
+        get_element(get(temperature), i);
       if constexpr (ThermodynamicDim == 1) {
```

Comment:
Can you also run clang-format on lines 145-161?

Reply:
I've run clang-format on the file so I think formatting should be fixed.